src/playable.rs

```diff
@@ -88,6 +88,10 @@ impl ListItem for Playable {
         self.as_listitem().play(queue)
     }
 
+    fn play_next(&mut self, queue: Arc<Queue>) {
+        self.queue(queue)
```

Comment:
Is there a reason this does not call `play_next` on the `ListItem` but `queue` instead?

Reply:
No idea how that got there, there's no reason for that. Fixing it now.